news/2023/2023-03-23-mascot-in-a-beatmap-art-contest.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "Mascot in a Beatmap Art Contest"
+date: 2023-03-23 16:00:00 +0000
+---
+
+Swap out your pen nibs and prepare your eyeballs — it's time for 2023's first art contest!
+
+![](https://assets.ppy.sh/contests/175/header.jpg)
+
+Our much-beloved art contests are back again for 2023, and we're starting things off with one of the themes of all time. So much of a theme in fact, that it is the least creatively named but most creative art contest we've ever held: the *Mascot in a Beatmap Art Contest*.
```

Comment:
> starting things off with one of the themes of all time

is this correct english? feels a bit weird.

Reply:
it is a meme

Reply:
okay i'll trust your meme knowledge today.

Reply:
it works well enough https://github.com/ppy/osu-wiki/pull/9012#discussion_r1146451748